### DISCUSSION TITLE
ERRORS-WAVE HEIGHT overflows normal value

### DISCUSSION THREAD

**Discussion by ZHHisme** [NOAA-EMC/WW3#689] [Q&A]
Why my WW3 result's WAVE HEIGHT  overflows normal value???
At the begining, HS seems normal, but about 1 hours later, the HS goes wrong... If anyone could figure the questions?? Thanks.

WW3 Result: (some pics)   At the begining,
![ww3-HS-at the begining](https://user-images.githubusercontent.com/103922752/165111726-981dd3ad-108b-4405-b645-bdf0b848ac4b.png)
![ww3-HS-wrong](https://user-images.githubusercontent.com/103922752/165111741-09685e9e-860a-4358-9020-9de7f6bb86a2.png)

Wind Field: (there is no wrong with my wind field)
![wind field](https://user-images.githubusercontent.com/103922752/165111765-301c6ca5-a124-4068-b1b9-690f03475d33.png)

Here are my input files:::::::::
_________ww3_strt.inp________________________________________________ww3_strt.inp____________________________ww3_strt.inp_________________
$ -------------------------------------------------------------------- $
$ WAVEWATCH III Initial conditions input file                          $
$--------------------------------------------------------------------- $
$ type of initial field ITYPE .
$
    3
$
$ ITYPE = 1 ---------------------------------------------------------- $
$ Gaussian in frequency and space, cos type in direction.
$ - fp and spread (Hz), mean direction (degr., oceanographic
$    convention) and cosine power, Xm and spread (degr. or m) Ym and
$    spread (degr. or m), Hmax (m) (Example for lon-lat grid in degr.).
$
$   0.10  0.01  270. 2  1. 0.5 1. 0.5 2.5
$    0.10  0.01  270. 2  0. 1000. 1. 1000. 2.5
    
    $  fp   sip     thm  ncos xm    six     ym     siy hmax
   0.15 0.02    240. 4  -10.E3  20.E3   20.E3 1.E8  1.00

$   0.10  0.01  270. 2  0. 1000. 1. 1000. 0.01
$   0.10  0.01  270. 2  0. 1000. 1. 1000. 0.
$
$ ITYPE = 2 ---------------------------------------------------------- $
$ JONSWAP spectrum with Hasselmann et al. (1980) direct. distribution.
$ - alfa, peak freq. (Hz), mean direction (degr., oceanographical
$   convention), gamma, sigA, sigB, Xm and spread (degr. or m) Ym and
$   spread (degr. or m)  (Example for lon-lat grid in degr.).
$   alfa, sigA, sigB give default values if less than or equal to 0.
$
$   0.0081  0.1  270.  1.0 0. 0. 1. 100. 1. 100.
$
$ ITYPE = 3 ---------------------------------------------------------- $
$ Fetch-limited JONSWAP
$ - No additional data, the local spectrum is calculated using the
$   local wind speed and direction, using the spatial grid size as
$   fetch, and assuring that the spectrum is within the discrete
$   frequency range.
$
$ ITYPE = 4 ---------------------------------------------------------- $
$ User-defined spectrum
$ - Scale factor., defaults to 1 if less than or equal 0.
$ - Spectrum F(f,theta) (single read statement)
$
$ -0.1
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 1 4 2 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 1 1 1 1 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 1 2 3 2 1 1 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 1 3 9 7 5 3 2 1 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 1 3 4 3 2 1 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 1 1 1 1 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$  0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
$
$ ITYPE = 5 ---------------------------------------------------------- $
$ Starting from calm conditions.
$ - No additional data.
$
$ -------------------------------------------------------------------- $
$ End of input file                                                    $
$ -------------------------------------------------------------------- $

__________________ww3_prnc.inp_________________________________________________ww3_prnc.inp______________________________________________
$
  'WND' 'LL' T T
$
$ Name of spatial dimensions------------------------------------------ $
$ NB: time dimension is expected to be called 'time' and must respect 
$     Julian or Gregorian calendar with leap day.
$
 longitude latitude time
$
$ Variables to use --------------------------------------------------- $
$
 u  v 
$
$ Additional time input ---------------------------------------------- $
$ If time flag is .FALSE., give time of field in yyyymmdd hhmmss format.
$
$   19680606 053000
$
$ Define data files -------------------------------------------------- $
$ The input line identifies the filename using for the forcing field.
$
  'wrfout_cpl_fnl_uv01_WW3.nc'
$
$ -------------------------------------------------------------------- $
$ End of input file                                                    $
$ -------------------------------------------------------------------- $

__________________________ww3_shel.inp________________________ww3_shel.inp___________________________ww3_shel.inp_______________________
$ -------------------------------------------------------------------- $
$ WAVEWATCH III shell input file                                       $
$ -------------------------------------------------------------------- $
$ Define input to be used with flag for use and flag for definition
$ as a homogeneous field; four input lines.
$ 1) Water levels
$ 2) Currents
$ 3) Winds
$ 4) Ice concentrations (cannot be homogeneous).
$
   F F
   F F
   T F
   F
   F       Assimilation data : Mean parameters
   F       Assimilation data : 1-D spectra
   F       Assimilation data : 2-D spectra.
$
$ Time frame of calculations ----------------------------------------- $
$ - Starting time in yyyymmdd hhmmss format.
$ - Ending time in yyyymmdd hhmmss format.
$
   20211201 000000
   20211203 000000
$
$---------------------------------------------------------
    1
$--------------------------------------------------------
$
$ Define output data ------------------------------------------------- $
$ Five output types are available (see below). All output types share
$ a similar format for the first input line:
$ - first time in yyyymmdd hhmmss format, output interval (s), and 
$   last time in yyyymmdd hhmmss format (all integers).
$ Output is disabled by setting the output interval to 0.
$
$ Type 1 : Fields of mean wave parameters
$          Standard line and line with flags to activate output fields
$          as defined in section 2.4 of the manual. The second line is
$          not supplied if no output is requested.
$                               The raw data file is out_grd.ww3, 
$                               see w3iogo.ftn for additional doc.
$          
$
   20211201 000000  3600  20211203 000000
$----------------------------------------------------------------
$ Output request flags identifying fields as in ww3_shel input and
$ section 2.4 of the manual.
$
 N
 HS DIR
$
$----------------------------------------------------------------
$
$ Type 2 : Point output
$          Standard line and a number of lines identifying the 
$          longitude, latitude and name (C*10) of output points.
$          The list is closed by defining a point with the name
$          'STOPSTRING'. No point info read if no point output is
$          requested (i.e., no 'STOPSTRING' needed).
$                               The raw data file is out_pnt.ww3, 
$                               see w3iogo.ftn for additional doc.
$
   20220317 000000 0   20220327 000000
$
$
$ Type 3 : Output along  track.
$          Flags for formatted input files.
$                         The data files are track_i.ww3 and
$                         track_o.ww3, see w3iotr.ftn for ad. doc.
$
   20080522 000000   0  20080525 000000
$     T
$
$ Type 4 : Restart files (no additional data required).
$                               The data file is restartN.ww3, see
$                               w3iors.ftn for additional doc.
$
   20220325 000000   0  20220413 000000
$
$ Type 5 : Boundary data (no additional data required).
$                               The data file is nestN.ww3, see
$                               w3iobp.ftn for additional doc.
$
   20080522 000000   0  20080525 000000
$ Type 6 : Whatever
   20080522 000000   0  20080525 000000
$
$ Testing of output through parameter list (C/TPAR) ------------------ $
$    Time for output and field flags as in above output type 1.
$
$  19680606 014500
$    T T T T T  T T T T T  T T T T T  T
$
$ Homogeneous field data --------------------------------------------- $
$ Homogeneous fields can be defined by a list of lines containing an ID
$ string 'LEV' 'CUR' 'WND', date and time information (yyyymmdd
$ hhmmss), value (S.I. units), direction (current and wind, oceanographic
$ convention degrees)) and air-sea temparature difference (degrees C).
$ 'STP' is mandatory stop string.
$
$  'LEV' 19680606 010000    0.0
$  'CUR' 19680606 073125    2.0    25.
$  'WND' 20010701 000000   20.    145.    2.0
$  'WND' 19680606 030000   20.    270.  -10.0
   'STP'
$
$ -------------------------------------------------------------------- $
$ End of input file                                                    $
$ -------------------------------------------------------------------- $

__________________________ww3_ounf.inp________________________ww3_ounf.inp___________________________ww3_ounf.inp______________________
$ -------------------------------------------------------------------- $
$ WAVEWATCH III Grid output post-processing                            $
$--------------------------------------------------------------------- $
$ First output time (yyyymmdd hhmmss), increment of output (s), 
$ and number of output times.
$
  20211201 000000  3600.  99999
$
$ Fields requested --------------------------------------------------- $
$
$ Output request flags identifying fields as in ww3_shel.inp. See that
$ file for a full documentation of field output options. Namelist type
$ selection is used here (for alternative F/T flags, see ww3_shel.inp).
$
$ DPT CUR WND AST WLV ICE IBG D50 IC1 IC5 HS LM T02 T0M1 T01 FP DIR SPR
$ DP HIG EF TH1M STH1M TH2M STH2M WN PHS PTP PLP PDIR PSPR PWS PDP
$ PQP PPE PGW PSW PTM10 PT01 PT02 PEP TWS PNR UST CHA CGE FAW TAW TWA WCC
$ WCF WCH WCM SXY TWO BHD FOC TUS USS P2S USF P2L TWI FIC ABR UBR BED
$ FBB TBB MSS MSC DTD FC CFX CFD CFK U1 U2 
$
 N
 HS DIR
$
$--------------------------------------------------------------------- $
$ netCDF version [3,4]
$        and variable type 4 [2 = SHORT, 3 = it depends , 4 = REAL]
$ swell partitions [0 1 2 3 4 5]
$ variables in same file [T] or not [F] 
$
 3 4
 0 1 
 T
$
$ -------------------------------------------------------------------- $
$ File prefix
$ number of characters in date [4(yearly),6(monthly),8(daily),10(hourly)]
$ IX and IY ranges [regular:IX NX IY NY, unstructured:IP NP 1 1]
$
 0413.         
 6
$
$ ----------------- FOR SMC GRID ONLY ---------------------------------$
$ SMC output type:
$    1 = Flat points file (1D sea point array)
$    2 = Regular gridded (2D lat/lon array over region)
$
$  1
$
$ For SMC output, the IX/IY range line is replaced with a domain
$ lat/lon range and `cellfac` parameter for SMC type 2 output.
$ First/Last lat/lon can be set to -999.9 to use edge of SMC grid.
$
$ For SMC type 1 output, only those points within the specified 
$ lat/lon ranges will be extracted:
$
$   first_lon, first_lat, last_lon, last_lat
$
$ For type 2 output, the range is used in conjunction with a fifth
$ integer `cellfac` parameter to specify the regular output grid
$ to area average the SMC grid to. In this case, the output grid will
$ be aligned to nearest largest SMC grid points within the selected 
$ region. Therefore, the output grid start and end lat/lons may differ 
$ slightly from what is requested. In order to obtain a fully populated
$ regular grid the extents specified should encompass the SW corner of
$ the bottom left cell, and NE corner of the top right cell required.
$ `cellfac` is an integer value that selects the target grid cell size
$ as multiple of the smallest SMC grid cell. It must be a power of 2.
$ e.g. 1 = highest resolution, 2 = half resolution of smallest cell, 
$ 4 = 1/4 res of smallest cell, etc.
$
$   first_lon, first_lat, last_lon, last_lat, cellfac
$ 
$ Example: Extract high resolution data for U.K.:
$
$ -13.50  46.85  5.50  61.0  1
$
$
$ ---------------- FOR NON-SMC GRIDS --------------------------------- $
$ IX, IY range:
 1 1000000 1 1000000 
$
$ For each field and time a new file is generated with the file name
$ ww3.date_xxx.nc , where date is a conventional time indicator with S3
$ characters, and xxx is a field identifier.
$
$ -------------------------------------------------------------------- $
$ End of input file                                                    $
$ -------------------------------------------------------------------- $

**Comment by aliabdolali** [comment]
@ZHHisme have you checked CFL and your time steps are defined according to it?
see https://github.com/NOAA-EMC/WW3/wiki/FAQs-page#CFL-condition
